docs/maven-tutorial.md

```diff
@@ -0,0 +1,218 @@
+      <id>nexus</id>
+      <mirrorOf>*</mirrorOf>
+      <url>http://nexus:8081/content/groups/public</url>
+    </mirror>
+  </mirrors>
+  <profiles>
+    <profile>
+      <id>nexus</id>
+      <!--Enable snapshots for the built in central repo to direct -->
+      <!--all requests to nexus via the mirror -->
+      <repositories>
+        <repository>
+          <id>central</id>
+          <url>http://central</url>
+          <releases><enabled>true</enabled></releases>
+          <snapshots><enabled>true</enabled></snapshots>
+        </repository>
+      </repositories>
+     <pluginRepositories>
+        <pluginRepository>
+          <id>central</id>
+          <url>http://central</url>
+          <releases><enabled>true</enabled></releases>
+          <snapshots><enabled>true</enabled></snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+  </profiles>
+  <activeProfiles>
+    <!--make the profile active all the time -->
+    <activeProfile>nexus</activeProfile>
+  </activeProfiles>
+</settings>
+```
+
+This is the crucial file for the effort.  We're telling Maven to use only the
+repository at nexus:8081, which is our proxy.  More information on DNS
+resolution in OpenShift can be found
+[here](https://docs.openshift.com/enterprise/3.1/architecture/additional_concepts/networking.html)
+
+Create a Dockerfile in the wildfly-builder directory with a `FROM` entry that
+points to the builder image you normally use.  Add a `COPY` entry pointing to
+the `settings.xml` you created earlier.
+
+```
+FROM docker.io/openshift/wildfly-100-centos7:latest
+
+COPY settings.xml /usr/share/maven/conf/settings.xml
+COPY settings.xml /opt/app-root/src/.m2/settings.xml
+```
+
+> NOTE: Maven's default global settings are in `/usr/share/maven/conf`, but some images
+> specify a settings.xml file when it is called.  For example, our WildFly builder
+> image uses `mvn -S /opt/app-root/src/.m2/settings.xml`, so we overwrite that
+> file with our own.  You may have to do the same.
+
+From your project's base directory, build a new imagestream from the
+'wildfly-builder' directory.
+```bash
+oc new-build wildfly-builder/ --to 'wildfly-nexus-10'
+oc start-build wildfly-nexus-10 --from-dir wildfly-builder/
+```
+
+Wait for this build to finish.  Afterward, start up your app with the modified
+image.
+
+```bash
+oc new-app wildfly-nexus-10:latest~https://github.com/bparees/openshift-jee-sample.git`
+```
+
+On the Nexus page we mentioned earlier, navigate to
+`http://<Nexus IP>:8081/content/groups/public` to confirm that it has stored
+your application's dependencies.
+
+Now let's redeploy the Nexus server to make sure the persistance works.
+`oc deploy nexus --latest`
+
+Refresh your browser.  You should still have your dependencies downloaded.
+
+## See Also
+
+* [Networking Concepts in OpenShift](https://docs.openshift.com/enterprise/3.1/architecture/additional_concepts/networking.html)
```

Comment:
It was just one of the documents I used to create the tutorial, I'll go and find the Origin equivalent.  Thanks for the feedback!